src/main/java/org/apereo/openequella/tools/toolbox/api/OpenEquellaRestUtils.java

```diff
@@ -335,6 +338,67 @@ public boolean gatherAccessToken() {
     return cachedItems;
   }
 
+  public List<ParsedCollection> gatherCollections() throws Exception {
+    CloseableHttpClient httpclient = HttpClients.createDefault();
+    String url = Config.get(Config.OEQ_URL) + "/api/collection";
+    HttpGet http = new HttpGet(url);
+
+    http.addHeader("X-Authorization", "access_token=" + accessToken);
+    LOGGER.debug("Making the API call: {}", url);
+    HttpResponse response = httpclient.execute(http);
+    HttpEntity entity = response.getEntity();
+
+    int statusCode = response.getStatusLine().getStatusCode();
+
+    String respStr = EntityUtils.toString(entity);
+    LOGGER.trace(statusCode);
+    LOGGER.trace(respStr);
+    if (statusCode != 200) {
```

Comment:
are any 3XX status codes, which may be valid, possible here?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#Redirection_messages

Reply:
hmmm, I haven't experienced the need for a redirect against the REST API when properly auth'd.  I think for now we proceed, but could be an enhancement if adopters start hitting 3xx on the APIs.